content/docs/2.0/scalers/azure-storage-blob.md

```diff
@@ -17,13 +17,13 @@ triggers:
 - type: azure-blob
   metadata:
     blobContainerName: functions-blob # Required: Name of Azure Blob Storage container
-    blobCount: '5' # Optional. Amount of blobs to scale out on. Default: 5 blobs 
-    connection: STORAGE_CONNECTIONSTRING_ENV_NAME # Optional if TriggerAuthentication defined with pod identity or connection string authentication.
+    blobCount: '5' # Optional. Amount of blobs to scale out on. Default: 5 blobs
+    connectionFromEnv: STORAGE_CONNECTIONSTRING_ENV_NAME # Optional if TriggerAuthentication defined with pod identity or connection string authentication.
     blobPrefix:  # Optional. Prefix for the Blob. Use this to specifiy sub path for the blobs if required. Default : ""
     blobDelimiter: # Optional. Delimiter for identifying the blob Prefix. Default: "/"
 ```
 
-The `connection` value is the name of the environment variable your deployment uses to get the connection string. This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.
+The `connectionFromEnv` value is the name of the environment variable your deployment uses to get the connection string. This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.
```

Comment:
General remark for all scalers - Does this mean our auth parameter is renamed to this as well? I don't think so, right?

For example here, do I use `connection` or `connectionFromEnv` [here](https://github.com/kedacore/keda-docs/pull/241/files#diff-7cf10601f41d343322f1a34a3d4ef023R34)?

Reply:
yes, the auth parameters are not renamed. Those are defined under `### Authentication Parameters` in each scaler doc. 

So if you are specifying the setting on the metadata, it takes `FromEnv` for values that are resolved from the scale target env. If you are specifying them through a `TriggerAuthentication`, then there is no `FromEnv` because it wouldn't make sense there as the `TriggerAuthentication` already defines where to resolve the value from, right?

The confusing thing about the linked example is that it shows either using `connectionFromEnv` or podIdentity. to use TriggerAuthentication in that example it would be 


```yaml
apiVersion: keda.sh/v1alpha1
kind: TriggerAuthentication
metadata:
  name: keda-trigger-auth-for-blob
spec:
  secretTargetRef:
  - parameter: connection
    name: my-secret-object
    key: AZURE_STORAGE_CONNECTION
```